xyz.mx_moment.moment.yaml

```diff
@@ -0,0 +1,783 @@
+id: xyz.mx_moment.moment
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15-21.08'
+command: moment
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --socket=wayland
+- --socket=pulseaudio
+- --device=dri
+- --talk-name=org.freedesktop.Notifications
+rename-icon: moment
+rename-desktop-file: moment.desktop
+rename-appdata-file: moment.metainfo.xml
```

Comment:
Ideally, these should happen upstream.

Reply:
Are you suggesting that the files should use the ID format upstream too?

Reply:
The original files should use the ID format. It's not a blocker, just mentioning that `rename` options are workarounds, not solutions.